target/linux/ipq40xx/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq4029-insect-common.dtsi

```diff
@@ -131,10 +131,14 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		mac_address: mac-address@66 {
-			compatible = "mac-base";
-			reg = <0x66 0x6>;
-			#nvmem-cell-cells = <1>;
+		nvmem-layout {
+			compatible = "fixed-layout";
+
```

Comment:
#address-cells = <1>; and #size-cells = <1>; are needed here I think.

Reply:
Rather, they should be moved.

Reply:
Good catch. Works without them, but `dtc` will surely complain underneath.

Reply:
Fixed.